pkg/build/gobuild.go

```diff
@@ -58,6 +60,20 @@ const (
 	defaultAppFilename = "ko-app"
 )
 
+// DefaultBaseImage returns the default base image us
+func DefaultBaseImage() (ref string, err error) {
+	if !viper.IsSet("defaultBaseImage") {
```

Comment:
Does this take into account `KO_CONFIG_PATH`?

Reply:
yep, I've added a test about that.